### ISSUE TITLE
write vignette

### ISSUE THREAD

**Issue by wcornwell** [bobdouma/kappa_multinomial#4]
Probably what is currently example.r can be pretty easily fleshed out into a proper vingette

**Comment by bobdouma** [comment]
yes, we could do so. Although I would need to think a bit on what is really needed and what is redundant
